.lando.yml

```
@@ -7,6 +7,22 @@ config:
   database: mariadb:10.4
   xdebug: false
 services:
+  # Replaces default lando MariaDB image with arm64 (Apple M1) compatible one.
+  # TODO: remove database service overrides once lando is updated.
+  database:
+    type: compose
+    services:
+      image: mariadb:latest
```

I found that using `10.4` here also worked.  Maybe that would be a "safer" option?
```suggestion
      image: mariadb:10.4
```

---

I will test this

---

FWIW using `mariadb:10.4` here is _NOT_ the same thing as the `database: mariadb:10.4` line (line 7).  Lando uses [`bitnami/mariadb`](https://hub.docker.com/r/bitnami/mariadb/) docker images by default which don't currently support arm64, which is why it crash on M1 Macs.  `mariadb:10.4` here is actually referencing the [default Docker mariadb image](https://hub.docker.com/_/mariadb), which does support arm64.

---

This change also removes the warnings I was seeing.